app/governance/page.tsx

```diff
@@ -1,5 +1,97 @@
-import AnimatedBackground from "@/components/animated_background/animatedBackground";
+"use client";
+import Tabs from "@/components/tabs/tabs";
+import { useState } from "react";
+
+import Container from "@/components/container/container";
+import { Proposal } from "@/hooks/gov/interfaces/proposal";
+import { ProposalModal } from "./proposalModal";
+import useProposals from "@/hooks/gov/useProposals";
+import ProposalTable from "./components/ProposalTable/ProposalTable";
+import Table from "@/components/table/table";
+import styles from './gov.module.scss';
+import Text from "@/components/text";
+import Spacer from "@/components/layout/spacer";
+import Button from "@/components/button/button";
+
 
 export default function GovernancePage() {
-  return <div>{/* <AnimatedBackground /> */}</div>;
-}
+  const { proposals } = useProposals({ chainId: 7700 });
+  //console.log(proposals);
+  
+
+  
+  const sorted_proposals = proposals.sort((a: Proposal, b: Proposal) => b.proposal_id - a.proposal_id);
+
+
+  const [activeTab, setActiveTab] = useState("All");
+
+  function switchActiveTab(activeTab: string) {
+    setActiveTab(activeTab);
```

Comment:
You can just setActive tab directly there's no need to create a dedicated function just for that